examples/react-tsx/.eslintignore

```diff
@@ -0,0 +1,3 @@
+**/node_modules/**
+**/bin/**
+**/lib/**
```

Comment:
there is no lib (and bin) folder in your example. So you don't need to exclude them.

Reply:
done :)